src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -84,6 +100,9 @@
         @Inject(optional=true) @Named("neverReplaceExistingPlugins")
         boolean neverReplaceExistingPlugins;
 
+        @Inject(optional = true) @Named("pluginEvaluationOutcome")
```

Comment:
Can this be provided by good old `-D...` properties? I would like to stay away from the config *.groovy binding insanity.